sdk/cognitivelanguage/ai-language-text/test/node/customTest.spec.ts

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assertEnvironmentVariable, isPlaybackMode, Recorder } from "@azure-tools/test-recorder";
+import { AnalyzeBatchActionNames, AzureKeyCredential, TextAnalysisClient } from "../../src";
+import { matrix } from "@azure/test-utils";
+import { Context, Suite } from "mocha";
+import { AuthMethod, createClient, startRecorder } from "../public/utils/recordedClient";
+import createAuthoringClient, { TextAuthoringClient } from "@azure/ai-language-textauthoring";
+import {
+  cleanupCustomTestResource,
+  createCustomTestProject,
+} from "../public/utils/customTestHelpter";
+import { assertActionsResults } from "../public/utils/resultHelper";
+import { expectation1, expectation2, expectation4 } from "../public/expectations";
+matrix([["APIKey"]] as const, async (authMethod: AuthMethod) => {
```

Comment:
Let's make sure to add AAD before we merge.
```suggestion
matrix([["APIKey", "AAD"]] as const, async (authMethod: AuthMethod) => {
```